Insight_Project_Framework/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM continuumio/miniconda3
+USER root
+
+RUN apt-get update -y
+RUN apt-get install -y gcc
+
```

Comment:
You can try something like the following: 
````
RUN apt-get install -y \
          gcc \
          poppler-utils \
          tesseract-ocr \
          libtesseract-dev \
          libcurl4-openssl-dev libssl-dev python-dev \
          curl \
          python3-pycurl \
 && apt-get autoremove -y \
 && apt-get clean \
 && apt-get autoclean \
 && rm -rf /var/lib/apt/lists/*
````
This will save some "RUN" command; the last 4 lines is to clean up the install and make the docker file smaller.

Reply:
updated.